### DISCUSSION TITLE
alephium (ALPH)

### DISCUSSION THREAD

**Discussion by altinsoft** [coinfoundry/miningcore#1139] [General]
Hi,

https://coinmarketcap.com/tr/currencies/alephium/

Coin : alephium (ALPH)

How can we create a template for this coin. Please help me.

Thank you.

**Comment by altinsoft** [comment]
@oliverw please help me. Thank you..

**Reply by oliverw** [reply]
> @oliverw please help me. Thank you..

Nope.

**Comment by sven-hash** [comment]
Is there some info about supporting this coin ?

Here some documentation on the pool software from Alephium https://github.com/alephium/mining-pool

**Reply by LoulouCrypto** [reply]
Very interested too !

**Comment by acidtib** [comment]
https://wiki.alephium.org/mining/pool-mining-guide

**Reply by LoulouCrypto** [reply]
Interesting and i'm running it, but there is not a lot of docs 

And thé interest is to have all on thé same pool software

**Comment by oliverw** [comment]
https://github.com/oliverw/miningcore/discussions/1466
